apps/public-data/src/public-data/public-data.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { DataSource, Repository } from 'typeorm';
+import { PostCategory } from './entities/post-category.entity';
+
+@Injectable()
+export class PublicDataService {
+  public constructor(
+    private dataSource: DataSource,
+    @InjectRepository(PostCategory)
+    private postCategoryRepository: Repository<PostCategory>,
+  ) {}
+
+  public async getPostData(): Promise<PostCategory[]> {
+    const queryRunner = this.dataSource.createQueryRunner();
+    await queryRunner.startTransaction();
+    try {
+      const postDatas = await this.postCategoryRepository.find();
+      await queryRunner.commitTransaction();
+      return postDatas;
+    } catch (err) {
+      await queryRunner.rollbackTransaction();
+      throw err;
+    } finally {
+      await queryRunner.release();
+    }
+  }
+
+  public async getOnePostData(id: number): Promise<PostCategory> {
+    const queryRunner = this.dataSource.createQueryRunner();
+    await queryRunner.startTransaction();
+    try {
+      const postData = await this.postCategoryRepository.findOne({ where: { id: id } });
```

Comment:
object shorthand 문법을 사용하지 않으면 린트에 걸리게 하면 좋을 것 같습니다.
```suggestion
      const postData = await this.postCategoryRepository.findOne({ where: { id } });
```

Reply:
해당 수정 부는 추후 config 성의 ISSUE로 새로운 PR에 작업해야 할 것 같습니다!